sdcm/tester.py

```diff
@@ -1952,6 +1954,55 @@ def get_cluster_k8s_eks(self, n_k8s_clusters: int):
             self.monitors = NoMonitorSet()
             self.monitors_multitenant = [self.monitors]
 
+    def get_cluster_cloud(self, loader_info, db_info):
+        cloud_api_client = ScyllaCloudAPIClient(
+            api_url=self.params.get('cloud_api_url'),
+            auth_token=self.params.get('cloud_api_token'),
+            raise_for_status=True)
+
+        user_prefix = self.params.get('user_prefix')
+
+        if db_info['n_nodes'] is None:
+            n_db_nodes = self.params.get('n_db_nodes')
+            if isinstance(n_db_nodes, int):
+                db_info['n_nodes'] = [n_db_nodes]
+            elif isinstance(n_db_nodes, str):
+                db_info['n_nodes'] = [int(n) for n in n_db_nodes.split()]
+            else:
+                self.fail('Unsupported parameter type: {}'.format(type(n_db_nodes)))
+
+        if loader_info['n_nodes'] is None:
+            n_loader_nodes = self.params.get('n_loaders')
+            if isinstance(n_loader_nodes, int):
+                loader_info['n_nodes'] = [n_loader_nodes]
+            elif isinstance(n_loader_nodes, str):
+                loader_info['n_nodes'] = [int(n) for n in n_loader_nodes.split()]
+            else:
+                self.fail('Unsupported parameter type: {}'.format(type(n_loader_nodes)))
+
+        self.log.info("Creating Scylla Cloud cluster")
+        self.db_cluster = ScyllaCloudCluster(
+            cloud_api_client=cloud_api_client,
+            user_prefix=user_prefix,
+            n_nodes=db_info['n_nodes'][0],
+            params=self.params,
+            region_names=self.params.get('cloud_region'),
+            add_nodes=True)
+
+        if loader_info['n_nodes'][0] > 0:
+            self.log.info("Creating Scylla Cloud loader set")
+            self.loaders = LoaderSetCloud(
+                cloud_api_client=cloud_api_client,
+                user_prefix=user_prefix,
+                n_nodes=loader_info['n_nodes'][0],
+                params=self.params,
+                region_names=self.params.get('cloud_region'))
+        else:
+            self.loaders = None
+
+        # monitors are not needed as Scylla Cloud has built-in monitoring
```

Comment:
this is something we should raise a task for, how we are gonna connect our monitor to the scylladb-cloud one.

previously we did federation on Prometheus level, question is how we can do it via the API